pkg/controller/ksmtuned/controller.go

```diff
@@ -82,6 +82,11 @@ func (c *Controller) OnChange(key string, kt *ksmtunedv1.Ksmtuned) (*ksmtunedv1.
 		return kt, nil
 	}
 
+	// process merge across nodes changed
+	if err := c.Ksmtuned.ToggleMergeAcrossNodes(kt.Spec.KsmtunedParameters.MergeAcrossNodes); err != nil {
+		return kt, err
+	}
+
```

Comment:
should we only config this when the ksmd is running?

Reply:
when ksmd running, pages are shared, so unable to config merge across nodes.